### PR TITLE
Add support to link student with URL with add command

### LINKED ISSUE #112: Include ability to add link for student in 'add' command

### PR DESCRIPTION
**Pull request by tanjiajiajun**
Fixes #112 

- Removed `link` command and its corresponding parser
- Updated `add` command and its parser

### CODE REVIEW

File: src/main/java/educonnect/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -43,15 +46,15 @@ public class AddCommandParser implements Parser<AddCommand> {
     public AddCommand parse(String args) throws ParseException {
```
Comment (fy17ohhh) [inline src/main/java/educonnect/logic/parser/AddCommandParser.java:46]: Could consider SLAP Hard in the parse method

File: src/main/java/educonnect/ui/HelpWindow.java [hunk 2]
```diff
@@ -26,7 +26,6 @@ public class HelpWindow extends UiPart<Stage> {
             + "'find' - Finds students whose names contain any of the given keywords\n"
             + "'help' - Brings up the help page\n"
             + "'list' - Shows a list of all students in EduConnect\n"
-            + "'link' - Links a website URL to a student\n"
```
Comment (fy17ohhh) [inline src/main/java/educonnect/ui/HelpWindow.java:29]: Glad to see that the HelpWindow is also update:))

**Review by fy17ohhh — COMMENTED** [review]
Code functionality LGTM:) Nicely done.
Could also consider improving readability.

**Review by marclamp — COMMENTED** [review]
LGTM :)

**Pull request merged**